pipelines/skylab/build_indices/Macaque.json

```diff
@@ -0,0 +1,8 @@
+{
+"BuildIndices.annotations_gtf":"gs://fc-c40ec8a8-d60f-42f7-be36-3986b475190a/Macaque/genomic.gtf",
+"BuildIndices.biotypes":"gs://fc-df68cb43-8c48-401b-9ef1-7cbb3acc788d/Biotypes.tsv",
+"BuildIndices.genome_fa":"gs://fc-c40ec8a8-d60f-42f7-be36-3986b475190a/Macaque/GCF_003339765.1_Mmul_10_genomic.fna",
```

Comment:
i've never heard of a .fna file. is this just a regular fasta file? or was this a typo?

Reply:
The extension *.fna just means fasta nucleic acid while *.fa or *.fasta formats contain other DNA-related information